libraries/AP_HAL_ChibiOS/hwdef/luminousbee5/hwdef-bl.dat

```diff
@@ -26,7 +29,13 @@ I2C_ORDER I2C1
 define HAL_I2C_INTERNAL_MASK 0
 
 # order of UARTs (and USB) for bootloading
-SERIAL_ORDER OTG1 UART7
+SERIAL_ORDER USART2 OTG1
```

Comment:
this order doesn't matter

Reply:
We had changed this for an experimental feature .We would like the bootloader to give priority to USART2 , corresponding to ESP8266 before USB .
So far this has not affected other bootloader operations and kept our feature working .